contracts/Marketplace.sol

```diff
@@ -30,6 +34,10 @@ contract Marketplace is Collateral, Proofs {
     collateral = _collateral;
   }
 
+  function myRequests() public view returns (RequestId[] memory) {
```

Comment:
Will this cause an implicit copy from `storage` to `memory`?

Reply:
Yes, but that is not so bad; it is a view function that is never called in a transaction.